docs/2_2_common_schema.adoc

```diff
@@ -384,7 +385,7 @@ in other words, use the connection equation (in this case the `relativeQuantity`
 see below,
 has to be taken into account in order to determine whether `offset` shall or shall not be utilized):_
 
-`factor(v1)*v1 + offset(v1) = factor(v2)*v2 + offset(v2)`
+`factor(v1) * v1 + (if relativeQuantity(v1) then 0 else offset(v1)) = factor(v2) * v2 + (if relativeQuantity(v2) then 0 else offset(v2))`
```

Comment:
Would you connect a relative quantity to an absolute quantity?
I.e. can v1 has relative quantity while v2 hasn't?

Reply:
@KarlWernersson: Does not seem to make sense. 
Please suggest a fix for FMI 3, and I will port it to FMI 2.0.2

Reply:
Should we add:
Where` relativeQuantity(v1) == relativeQuantity(v2)` is required

Reply:
yes that could be a minimum fix. Sorry I have not had time today to do any fixes

Reply:
@KarlWernersson : I committed that change.  could you please review or propose a better wording?
If accepted, I will create a PR to add this change to 3.0 also.